### PR TITLE
Fix/7657 table style

### LINKED ISSUE #7657: Rocket Insights UI validation
**Context**

I opened this issue to have all UI needed changes to be done in one place, so @wp-media/qa-team @piotrbak Plz update this issue's body to have all the points u have in mind and we can validate and move forward here:

**Rocket Insights list:**
- [x] Urls list table should have the color `#F9FAFB` as a background, now it's grey.
- [x] The global score about the listing table should be outside the border of the table. 
- [x] When doing any change to the urls (retest or add new url or remove) we should refresh the global score above the listing table.
- [x] Under performance summary, we should have a separator (paid)
- [x] After URL is processed, retest is dimmed while it should be active (paid and free)
- [x] When adding URL, while it's inprogress, retest should be dimm like see report => now it's active

Image

- The Add Page + button is active, also the limit was reached (tested on Paid)  ===> will be handled on a separate GH https://github.com/wp-media/wp-rocket/issues/7663
- [x] The dot after the page title in the Performance Summary section should be aligned to the middle, not to the bottom. Also, the left/right margins should be bigger
Image
Image
 
- [x] The separator after the score doesn't have correct margins and it's not correct size 
 
Image
Image 

- [x] After fresh install of free, and after home score is done, global score isnot automatically updated till manual refresh is done => may be it will be handled with 3rd point in the list 
- [x] The section with the list of URLs should have the border-radius
- [x] Re-test and see report should be bold and have the same font as in UI 
actual

Image

UI

Image

**Rocket Insights license banner:** 
- [x] Title font size is 24px, but should be 18px. 
- [x] Subtitle’s color is dark grey, but should be black (#0C0F1F). All texts (title, description, bullet points, big price) should be #0C0F1F. 
- [x] "Launch Offer" should be semibold and its color updated from #121116 → #0C0F1F.
- [x] "/month*" text should be positioned slightly higher; color should change from #777777 → #738191. 
- [x] "(excl. VAT)" should change from #738191 → #0C0F1F.
- [x] ~old price~ color should change from #999999 → #A9B2BC. ❌ 
- [x] "*Billed monthly [...]" bottom text should be #738191.
- [x] Bullet points should align with the top margin of the price box. Price box should be wider, with more padding between it and the subtitle. Also, there’s too much padding between the "Get Started" button and the margin.
- [x] Some key words in each bullet should be in bold. Checkmarks should appear inside blue background circles.
Actual:

Image

**Mostafa's comments:**
- Price font `Gotham Medium` not used in the plugin if we want to add it please send me the font files.
- `Some key words in each bullet should be in bold`, this is not possible anymore because the list coming from the API without HTML tags  <strong>

UI 

Image

**Global score in dashboard**
 - [x] The Global Score needs to be udpated. The whole circle should be smaller and bolder, also the size and weight of the font doesn't match. The button doesn't have correct paddings, the margin above the button is not the same.

Image

**Expected behavior**

UI Should be exactly the same as the last figma file here: https://www.figma.com/design/q8796TwdkNRsRIUZes0u28/WP-Rocket-Nicolas-s-drafts?node-id=269-4814&m=dev

### PR DESCRIPTION
**Pull request by mostafa-hisham**
# Description

Fixes #7657 

## Type of change

- [ ] New feature (non-breaking change which adds functionality).
- [x] Bug fix (non-breaking change which fixes an issue).
- [ ] Enhancement (non-breaking change which improves an existing functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as before).
- [ ] Sub-task of #(issue number)
- [ ] Chore
- [ ] Release

### What was tested

UI changes

### How to test

go to tab

### Documentation

no
### New dependencies

no
### Risks

no
# Mandatory Checklist

## Code validation

- [x] I validated all the Acceptance Criteria. If possible, provide screenshots or videos.
- [x] I triggered all changed lines of code at least once without new errors/warnings/notices.
- [x] I implemented built-in tests to cover the new/changed code.

## Code style

- [x] I wrote a self-explanatory code about what it does.
- [x] I protected entry points against unexpected inputs.
- [x] I did not introduce unnecessary complexity.
- [x] Output messages (errors, notices, logs) are explicit enough for users to understand the issue and are actionnable.

## Unticked items justification

*If some mandatory items are not relevant, explain why in this section.*

# Additional Checks
- [ ] In the case of complex code, I wrote comments to explain it.
- [ ] When possible, I prepared ways to observe the implemented system (logs, data, etc.).
- [ ] I added error handling logic when using functions that could throw errors (HTTP/API request, filesystem, etc.)

### CODE REVIEW

File: src/js/global/ajax.js [hunk 1]
```diff
@@ -323,6 +324,23 @@ document.addEventListener('DOMContentLoaded', function() {
...
+	function isOnRocketInsights() {
+		const urlParams = new URLSearchParams(window.location.search);
+		return urlParams.get('page') === 'wprocket' && window.location.hash === '#rocket_insights';
+	}
+
+	function updateGlobalScoreRow(response){
+		if ( isOnRocketInsights() ) {
+			const $tableGlobalScore = $('.wpr-pma-urls-table .wpr-global-score');
+			if ($tableGlobalScore.length > 0){
+				console.log(response.data.global_score_data.row_html);
```
Comment (remyperona) [inline src/js/global/ajax.js]:
```suggestion
```

**Review by Miraeld — APPROVED** [review]

File: src/js/global/ajax.js (hunk 1 above)
Comment (Miraeld) [reply src/js/global/ajax.js]: Fixed with latest commit

**Comment by jeawhanlee** [comment]
Need to fix conflicts here.
